rpmlint/descriptions/DBusServicesCheck.toml

```diff
@@ -0,0 +1,8 @@
+AUDIT_BUG_URL = "https://en.opensuse.org/openSUSE:Package_security_guidelines#audit_bugs"
```

Comment:
Please create a new `rpmlint/descriptions/Variables.toml` file and mode there `AUDIT_BUG_URL` and remove all other definitions of it `git grep 'AUDIT_BUG_URL = '`.